### DISCUSSION TITLE
Where to file issues regarding the learning equality website ?

### DISCUSSION THREAD

**Discussion by ForkMeMaybe** [learningequality/kolibri-design-system#842] [Q&A] [answered]
I am new to open source and wanted to contribute to this organization since i can personally connect with your goals. So i was just browsing around your website "https://learningequality.org/" and found an issue. But since there are so many repositories, i am not sure where to file issues regarding the website.

**Comment by MisRob** [comment, marked as answer]
Hi @ForkMeMaybe, thank you for reaching out. We haven't yet come to conclusion. Feel free to open to `kolibri` repo for now and we will organize.

**Comment by MisRob** [comment]
@ForkMeMaybe 

> I am new to open source and wanted to contribute to this organization since i can personally connect with your goals

Welcome and thank you! Glad to have you here.
